lib/PTML/Doxygen.cpp

```diff
@@ -265,8 +265,8 @@ gatherArgumentComments(const model::Binary &Binary,
   static constexpr std::string_view Keyword = "\\param ";
```

Comment:
Let this thread serve as a reminder of
https://github.com/revng/revng/pull/358#discussion_r1513282116

cc: @aleclearmind

Reply:
Done (see the last commit).